src/js/expanderExpanded.js

```diff
@@ -54,7 +57,7 @@ function mdExpanderExpandedDirective($mdUtil, $animateCss) {
       })
       .start()
       .then(function () {
-        if (isHeightSet !== undefined) {
+        if (isHeightSet === true) {
```

Comment:
Personally i like to add the true/false so anyone else reading the code can easily see the type, either way is correct.